app/routes/press-releases/overview/published.js

```diff
@@ -1,4 +1,22 @@
 import Route from '@ember/routing/route';
+// eslint-disable-next-line ember/no-mixins
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class PressReleasesOverviewPublishedRoute extends Route {
-}
+
+export default class PressReleasesOverviewPublishedRoute extends Route.extend(DataTableRouteMixin) {
+  modelName = 'press-release';
+
+  mergeQueryOptions(params) {
+    const queryParams = {
+      sort: params.sort,
+      include: 'publication-event',
+      filter: {
+        ':has:publication-event': 'yes',
```

Comment:
Aangezien we een filter gaan toepassen op de startdatum van het publication-event is deze filter overbodig.